lib/v2/datasource/remote/model/balance_model.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:seeds/utils/double_extension.dart';
+
+class BalanceModel {
```

Comment:
existing code. moved here

Reply:
yeah that was a doub I have if we also must delete each model we move to V2 in the original file.

Reply:
if its possible to remove, Yes